.travis.yml

```diff
@@ -28,6 +28,7 @@ install:
   - conda config --append channels conda-forge
   - conda config --append channels $PCDS_CHANNEL
   - conda config --append channels pcds-tag
+  - conda config --append channels pydm-tag
```

Comment:
Stuff like this makes me want to move sooner on filling our channel

Reply:
Yup. Noted in https://github.com/pcdshub/typhon/pull/45 as well